test/python/algorithms/test_vqe.py

```diff
@@ -17,6 +17,7 @@
 
 from functools import partial
 import numpy as np
+import scipy
```

Comment:
Given that you use `scipy.optimize`, you should import that directly.  Lower down, it only works as a side-effect that the `qiskit.algorithms` module imports `scipy.optimize` at some point - if that ever stops, then the test lower down will fail.

For example, try just opening a Python interpreter and doing
```python
>>> import scipy
>>> scipy.optimize.minimize
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'scipy' has no attribute 'optimize'
```

Reply:
Since only `scipy.optimize.minimize` is used, I just imported that function directly